source/documentation/contribute-to-rdoproject.html.md

```diff
@@ -0,0 +1,92 @@
+---
+author: chandankumar
+title: Let's Contribute to RDO Project
+---
+
+# Let's Contribute to RDO Project
+
+So contributing to RDO Project is hard, I think no! we need to follow some steps
+and we are done towards making first contribution to RDO Project.
+
+## Let's get prerequities ready.
+
+### Join the RDO Mailing List
+* Go to [RDO mailing list](https://www.redhat.com/mailman/listinfo/rdo-list) and enter your email
+  address and password and click on *subscribe*. Once done, Open your mail box, you will get an
+  confirmation email, click on the verification link and you are subscribed to the RDO List.
+* You can introduce yourself sending a mail to the mailing list. Tell us what are your interests
+  and how you will participate.
+
+### Join IRC channel #rdo on Freenode server.
+* All the development and communication related to RDO Project happens on #rdo IRC channel on
+  Freenode server. Click [this link](http://webchat.freenode.net/?channels=#rdo) to join the
+  same and feel free to say Hello!, we are always there to help.
+* Feel free to checkout the [IRC etiquette](https://www.rdoproject.org/community/irc-etiquette/)
+
+### Create a github account.
+Go to [Github Sign Up](https://github.com/join) and enter your username, email address and
+password and you are done. If you already have a github account. You can skip this step.
+
+### Sign to review.rdoproject.org
+Sign-in to [review.rdoproject.org](https://review.rdoproject.org/auth/login) using github account.
+
+### Add SSH key to review.rdoproject.org
+* Generate your SSH key, if you donot have.
+  Follow this link on [Generating a new SSH key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key)
+* Go to [ssh-key](https://review.rdoproject.org/r/#/settings/ssh-keys) and click on "Add Key" button
+  and copy the id_rsa.pub (SSH Key public key) and paste it there and then click on 'Add' button and you are done.
```

Comment:
We should add a small clarification: "and copy the contents of id_rsa.pub". It should be taken for granted, but just in case :)